src/foam/u2/view/ColumnConfigView.js

```diff
@@ -726,8 +713,16 @@ foam.CLASS({
             arr.push(childProps);
           }
         }
-        return arr;
+        if ( arr && arr.length > 0 )
+          return arr;
       }
+      if ( this.level === 0 ) {
+        if ( foam.Array.isInstance(this.rootProperty) )
+          return [[this.rootProperty[0]]];
+        else
+          return [this.rootProperty];
+      } else 
```

Comment:
No need for 'else' if you've already returned inn the 'if'.

Reply:
fixed